WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowCompletionViewController.swift

```diff
@@ -24,27 +23,33 @@ class BloggingRemindersFlowCompletionViewController: UIViewController {
 
     private let titleLabel: UILabel = {
         let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
```

Comment:
What do you think about having some convenience factory method on UILabel that sets both of these properties to true?

Reply:
It may make sense, but if we decided to do that, I'd consider doing it separately and also take into consideration standard `minimumScaleFactor` values for our labels.

Right now I'm content with having these values enabled on a label by label basis - less changes to take care of today.